pkg/controller/topology.go

```diff
@@ -134,19 +150,26 @@ func GenerateAccessibilityRequirements(
 	return requirement, nil
 }
 
+// aggregateTopologies returns all the supported topology values in the cluster that
+// match the driver's topology keys.
 func aggregateTopologies(
 	kubeClient kubernetes.Interface,
 	driverName string,
 	selectedNode *v1.Node) ([]topologyTerm, error) {
 
+	// 1. Determine topologyKeys to use for aggregation
 	var topologyKeys []string
 	if selectedNode == nil {
-		// TODO (verult) retry
+		// Immediate binding
+
+		// TODO (#144): use informers
 		nodeInfos, err := kubeClient.StorageV1beta1().CSINodes().List(metav1.ListOptions{})
 		if err != nil {
-			// We must support provisioning if CSINode is missing, for backward compatibility.
-			klog.Warningf("error listing CSINodes: %v; proceeding to provision without topology information", err)
-			return nil, nil
+			// If the topology feature in the external-provisioner is enabled,
+			// CSINode beta feature on K8s apiserver must also be enabled.
+			// We don't want to fallback and provision in the wrong topology if there's some temporary
+			// error with the API server.
+			return nil, fmt.Errorf("error listing CSINodes: %v", err)
```

Comment:
If we need to support running newer external provisioner on 1.13, then I think we need to disable topology by default.  Otherwise, provisioning on 1.13 will start to hit this error (if the driver also supports csi topology capability).